pkg/reconciler/cluster/cluster.go

```diff
@@ -70,14 +62,37 @@ func (c *Controller) reconcile(ctx context.Context, cluster *clusterv1alpha1.Clu
 		c.apiImporters[cluster.Name] = apiImporter
 	}
 
-	objs, err := c.apiresourceImportIndexer.ByIndex(LocationInLogicalClusterIndexName, GetLocationInLogicalClusterIndexKey(cluster.Name, logicalCluster))
-	if err != nil {
-		klog.Errorf("error in cluster reconcile: %v", err)
+	if success, err := c.manageSyncer(ctx, cluster, cfg); err != nil {
 		return err
+	} else if !success {
+		return nil // Don't retry
 	}
 
+	// Enqueue another check later
```

Comment:
why?

Reply:
Despite what github is telling you, L71-79 are unchanged. I have no idea what the original intent here was.

Reply:
Removed in latest revision. Definitely unnecessary.